1.0.0rc2: test_ts_to_array fails

When testing version 1.0.0 rc2 during build time, I see the following failure on the `test_ts_to_array` test:
```
_____________________________________ test_ts_to_array ______________________________________

generic_ts = <sunpy.timeseries.timeseriesbase.GenericTimeSeries object at 0x7fe084b4b9b0>

    def test_ts_to_array(generic_ts):
>       arr = generic_ts.to_array()

sunpy/timeseries/tests/test_timeseriesbase.py:612: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
sunpy/timeseries/timeseriesbase.py:576: in to_array
    return self.data.to_numpy(**kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self =                      intensity
2016-10-01 05:00:00   0.000000
2016-10-01 04:59:00   0.026177
2016-10-01 04:58:00   0.0...8
2016-09-30 05:03:00  -0.078459
2016-09-30 05:02:00  -0.052336
2016-09-30 05:01:00  -0.026177

[1440 rows x 1 columns]
name = 'to_numpy'

    def __getattr__(self, name):
        """After regular attribute access, try looking up the name
        This allows simpler access to columns for interactive use.
        """
    
        # Note: obj.x will always call obj.__getattribute__('x') prior to
        # calling obj.__getattr__('x').
    
        if (name in self._internal_names_set or name in self._metadata or
                name in self._accessors):
            return object.__getattribute__(self, name)
        else:
            if self._info_axis._can_hold_identifiers_and_holds_name(name):
                return self[name]
>           return object.__getattribute__(self, name)
E           AttributeError: 'DataFrame' object has no attribute 'to_numpy'

/usr/lib/python3/dist-packages/pandas/core/generic.py:4378: AttributeError
```
Pandas: 0.23.3
Astropy: 3.1.2
Python: 3.7.3
Numpy: 1.16.2

--- a/sunpy/timeseries/timeseriesbase.py
+++ b/sunpy/timeseries/timeseriesbase.py
@@ -567,7 +567,10 @@ class GenericTimeSeries:
         `~numpy.ndarray`
             If the data is heterogeneous and contains booleans or objects, the result will be of ``dtype=object``.
         """
-        return self.data.to_numpy(**kwargs)
+        if hasattr(self.data, "to_numpy"):
+            return self.data.to_numpy(**kwargs)
+        else:
+            return self.data.values
 
     def __eq__(self, other):
         """

